src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+import { ToDo } from '../../types/ToDo';
+import { DispatchContext } from '../../context/ToDoContext';
+
+interface Props {
+  plan: ToDo;
+}
+
+export const Todo: React.FC<Props> = ({ plan }) => {
+  const [newTitle, setNewTitle] = useState(plan.title);
```

Comment:
Use [destruction assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) for a plan object